### ISSUE TITLE
mesos logroate issues 

### ISSUE THREAD

**Issue by stevendborrelli** [CiscoCloud/mantl#1511]
- Git commit hash or branch: 1.1
- Cloud Environment: openstack

Mesos logration settings cause lots of incorrect archive files to show up. 
The current configuration is 

```
/var/log/mesos/* {
        daily
        rotate 7
        copytruncate
        missingok
        compress
        delaycompress
        notifempty
        minsize 1
}
```

Which generates the following log files after a while:

```
mesos-master.log
mesos-master.log-20160519.gz-20160522
mesos-master.log-20160519.gz-20160522-20160524
mesos-master.log-20160519.gz-20160522-20160524-20160526
mesos-master.log-20160519.gz-20160522-20160524-20160526-20160528
mesos-master.log-20160519.gz-20160522-20160524-20160526-20160528-20160530
mesos-master.log-20160519.gz-20160522-20160524-20160526-20160528-20160530-20160601
mesos-master.log-20160519.gz-20160522-20160524-20160526-20160528-20160530-20160601-20160603
mesos-master.log-20160519.gz-20160522-20160524-20160526-20160528-20160530-20160601-20160603-20160605
mesos-master.log-20160519.gz-20160522-20160524-20160526-20160528-20160530-20160601-20160603-20160605-20160607
mesos-master.log-20160520.gz-20160523
mesos-master.log-20160520.gz-20160523-20160525
mesos-master.log-20160520.gz-20160523-20160525-20160527
mesos-master.log-20160520.gz-20160523-20160525-20160527-20160529
mesos-master.log-20160520.gz-20160523-20160525-20160527-20160529-20160531
mesos-master.log-20160520.gz-20160523-20160525-20160527-20160529-20160531-20160602
mesos-master.log-20160520.gz-20160523-20160525-20160527-20160529-20160531-20160602-20160604
mesos-master.log-20160520.gz-20160523-20160525-20160527-20160529-20160531-20160602-20160604-20160606
mesos-master.log-20160521.gz-20160524
mesos-master.log-20160521.gz-20160524-20160526
mesos-master.log-20160521.gz-20160524-20160526-20160528
mesos-master.log-20160521.gz-20160524-20160526-20160528-20160530
mesos-master.log-20160521.gz-20160524-20160526-20160528-20160530-20160601
mesos-master.log-20160521.gz-20160524-20160526-20160528-20160530-20160601-20160603
mesos-master.log-20160521.gz-20160524-20160526-20160528-20160530-20160601-20160603-20160605
mesos-master.log-20160521.gz-20160524-20160526-20160528-20160530-20160601-20160603-20160605-20160607
mesos-master.log-20160522.gz-20160525
mesos-master.log-20160522.gz-20160525-20160527
mesos-master.log-20160522.gz-20160525-20160527-20160529
mesos-master.log-20160522.gz-20160525-20160527-20160529-20160531
mesos-master.log-20160522.gz-20160525-20160527-20160529-20160531-20160602
mesos-master.log-20160522.gz-20160525-20160527-20160529-20160531-20160602-20160604
mesos-master.log-20160522.gz-20160525-20160527-20160529-20160531-20160602-20160604-20160606
mesos-master.log-20160523.gz-20160526
mesos-master.log-20160523.gz-20160526-20160528
mesos-master.log-20160523.gz-20160526-20160528-20160530
mesos-master.log-20160523.gz-20160526-20160528-20160530-20160601
mesos-master.log-20160523.gz-20160526-20160528-20160530-20160601-20160603
mesos-master.log-20160523.gz-20160526-20160528-20160530-20160601-20160603-20160605
mesos-master.log-20160523.gz-20160526-20160528-20160530-20160601-20160603-20160605-20160607
mesos-master.log-20160524.gz-20160527
mesos-master.log-20160524.gz-20160527-20160529
mesos-master.log-20160524.gz-20160527-20160529-20160531
mesos-master.log-20160524.gz-20160527-20160529-20160531-20160602
mesos-master.log-20160524.gz-20160527-20160529-20160531-20160602-20160604
mesos-master.log-20160524.gz-20160527-20160529-20160531-20160602-20160604-20160606
mesos-master.log-20160525.gz-20160528
mesos-master.log-20160525.gz-20160528-20160530
mesos-master.log-20160525.gz-20160528-20160530-20160601
mesos-master.log-20160525.gz-20160528-20160530-20160601-20160603
mesos-master.log-20160525.gz-20160528-20160530-20160601-20160603-20160605
mesos-master.log-20160525.gz-20160528-20160530-20160601-20160603-20160605-20160607
mesos-master.log-20160526.gz-20160529
mesos-master.log-20160526.gz-20160529-20160531
mesos-master.log-20160526.gz-20160529-20160531-20160602
mesos-master.log-20160526.gz-20160529-20160531-20160602-20160604
mesos-master.log-20160526.gz-20160529-20160531-20160602-20160604-20160606
mesos-master.log-20160527.gz-20160530
mesos-master.log-20160527.gz-20160530-20160601
mesos-master.log-20160527.gz-20160530-20160601-20160603
mesos-master.log-20160527.gz-20160530-20160601-20160603-20160605
mesos-master.log-20160527.gz-20160530-20160601-20160603-20160605-20160607
mesos-master.log-20160528.gz-20160531
mesos-master.log-20160528.gz-20160531-20160602
mesos-master.log-20160528.gz-20160531-20160602-20160604
mesos-master.log-20160528.gz-20160531-20160602-20160604-20160606
mesos-master.log-20160529.gz-20160601
mesos-master.log-20160529.gz-20160601-20160603
mesos-master.log-20160529.gz-20160601-20160603-20160605
mesos-master.log-20160529.gz-20160601-20160603-20160605-20160607
mesos-master.log-20160530.gz-20160602
mesos-master.log-20160530.gz-20160602-20160604
mesos-master.log-20160530.gz-20160602-20160604-20160606
mesos-master.log-20160531.gz-20160603
mesos-master.log-20160531.gz-20160603-20160605
mesos-master.log-20160531.gz-20160603-20160605-20160607
mesos-master.log-20160601.gz
mesos-master.log-20160601.gz-20160604
mesos-master.log-20160601.gz-20160604-20160606
mesos-master.log-20160602.gz
mesos-master.log-20160602.gz-20160605
mesos-master.log-20160602.gz-20160605-20160607
mesos-master.log-20160603.gz
mesos-master.log-20160603.gz-20160606
mesos-master.log-20160604.gz
mesos-master.log-20160604.gz-20160607
mesos-master.log-20160605.gz
mesos-master.log-20160606.gz
mesos-master.log-20160607
```

**Comment by stevendborrelli** [comment]
it should probably just match on `/var/log/mesos-master.log` and `/var/log/mesos-agent.log` instead of `*`. Maybe we match `*.log`

**Comment by stevendborrelli** [comment]
Testing this on a prod system with log rotate.d settings: 

```
/var/log/mesos/*.log {
        daily
        rotate 7
        copytruncate
        missingok
        compress
        delaycompress
        notifempty
        minsize 1
}
```

**Comment by ryane** [comment]
fixed by #1529
